sdk/keyvault/keyvault-secrets/test/internal/challengeBasedAuthenticationPolicy.spec.ts

```diff
@@ -24,7 +24,7 @@ import { WebResource } from "@azure/core-http";
 // we will be able to unit test the insides in detail.
 
 describe("Challenge based authentication tests", () => {
-  const secretPrefix = `challengeAuth${env.KEY_NAME || "SecretName"}`;
+  const secretPrefix = `challengeAuthSecretName`;
```

Comment:
Thanks for doing this! This was hard to catch eh 😁 seems like some code that we accidentally left here from a while ago, as we based some of this code from Keys.